packages/dashboard/src/Dashboard.vue

```diff
@@ -284,11 +284,6 @@ export default class Dashboard extends Vue {
 
   async updated() {
     this.accounts = this.$store.state.portal.accounts;
-    if (this.$api && this.$route.path == "/") {
-      this.loadingAPI = false;
-    } else if (this.$route.path !== "/") {
-      this.loadingAPI = false;
-    }
```

Comment:
Removed this part because it doesn't make any sense, it sets the API Loading variable to false in all routes on the updated lifecycle, so the API read as undefined depends on its actual value of it, which it break the request